include/zephyr/drivers/clock_control/stm32_clock_control.h

```diff
@@ -428,6 +431,12 @@
 #define STM32_CKPER_ENABLED	1
 #endif
 
+#if DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(clk48), st_stm32_clock_mux, okay)
+/* Assuming the 48MHz clock is 48MHz */
+#define STM32_CK48_ENABLED	1
+#define STM32_CK48_FREQ		48000000
```

Comment:
Although it is named 48MHz clock, it can technically be another value so it would be safer to compute its value.
This computed value could then be used to throw an assert if not equal to 48MHz.

Reply:
We can imagine a get_ck48_frequency() function to return the clock freq depending on the clock source (CK48MSEL )  of the PLL48 to be part of the stm32_clock_control_get_subsys_rate (get_rate). In this case I do not see any difference between the _dev_dt_clk_freq_ and the _dev_actual_clk_freq_   
--> actually there is no more _dev_dt_clk_freq_  (except if we add a clock-frequency property to the "st,stm32-clock-mux")